validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -496,7 +507,8 @@ private AttesterDuties getAttesterDutiesFromIndexesAndState(
         ValidatorsUtil.getValidatorPubKey(state, UInt64.valueOf(validatorIndex)),
         CommitteeAssignmentUtil.get_committee_assignment(state, epoch, validatorIndex),
         (pkey, committeeAssignment) -> {
-          final UInt64 committeeCountPerSlot = get_committee_count_per_slot(state, epoch);
+          final UInt64 committeeCountPerSlot =
+              specProvider.at(state).getBeaconStateUtil().getCommitteeCountPerSlot(state, epoch);
```

Comment:
Probably should use `atEpoch(epoch)` here instead of `at(state)` as well as the state may be from an earlier epoch.